apps/RayCaster/inc/peripherals.h

```diff
@@ -0,0 +1,15 @@
+#ifndef CHIPEE_PERIPHERALS_H_
```

Comment:
The "CHIPEE" is intended to be changed to the app name (I forgot to do so when creating the C example, but the C++ example seems right).